### PR TITLE
Extract awk scripts to separate files

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Use IntelliJ IDEA Community Edition/Ultimate.
     * [Scala plugin](https://plugins.jetbrains.com/plugin/1347-scala) might be useful for editing UI tests.
 
 5. (optional) Install further non-bundled plugins from Marketplace:
+   * [AWK Support](https://plugins.jetbrains.com/plugin/17037-awk-support)
```
Comment (mkondratek) [inline CONTRIBUTING.md:34]: :shocked_pikachu:

File: scripts/enforce-indent-two-spaces-outside-java [hunk 2]
```diff
@@ -2,24 +2,11 @@
...
-  }
-
-  /^.+$/ { prev_non_empty = $0 }
-
-  END { exit exit_code }
-
-EOF
-)"
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+self_name=$(basename -s .sh "$0")
```
Comment (mkondratek) [inline scripts/enforce-indent-two-spaces-outside-java:6]: ok, so it means take current file name without the `.sh` suffix...

File: scripts/enforce-indent-two-spaces-outside-java [hunk 3]
```diff
@@ -2,24 +2,11 @@
...
 
 git ls-files \
   ':!*.flex' ':!gradlew' ':!*.java' ':!*.kt' ':!*.md' \
   ':!plugin-verifier.gradle' '!*.scala' ':!scripts/setup-multiroot-sandbox' \
   ':!*.svg' ':!testCommon/src/test/resources/*.sh' ':!*.xml' \
-   | xargs awk "$awk_script"
+   | xargs awk -f "$self_dir/$self_name.awk"
```
Comment (mkondratek) [inline scripts/enforce-indent-two-spaces-outside-java:12]: and here you refer to `scripts/enforce-indent-two-spaces-outside-java.awk` with it

File: scripts/enforce-indent-two-spaces-outside-java.awk [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env awk
```
Comment (mkondratek) [inline scripts/enforce-indent-two-spaces-outside-java.awk:1]: I do not understand this language XD looks fine

Comment (mkondratek) [reply scripts/enforce-indent-two-spaces-outside-java.awk:1]: nice to have it extracted!

**Review by mkondratek — APPROVED** [review]
🦆

File: scripts/enforce-indent-two-spaces-outside-java.awk (hunk 4 above)
Comment (PawelLipski) [reply scripts/enforce-indent-two-spaces-outside-java.awk:1]: `awk` is love, `awk` is life

**Pull request merged**